src/sections/product-main-template.liquid

```diff
@@ -209,7 +276,8 @@
 
                     </div>
                   {% endif %}
-
+                {% endunless %}
```

Comment:
this indentation doesn't look like it's matching up.  I've been looking for a viable formatter for liquid templates but so far haven't found one.

Reply:
make sure the unless/endunless match indentation and then children  have one more indentation level. It's a nightmare working through difficult to match elements.